subscriptions/subscriptions-impl/src/test/java/com/duckduckgo/subscriptions/impl/messaging/SubscriptionMessagingInterfaceTest.kt

```diff
@@ -656,6 +656,20 @@ class SubscriptionMessagingInterfaceTest {
         verify(subscriptionsManager).refreshAccessToken()
     }
 
+    @Test
+    fun whenProcessAndRemoveEmailSuccessAnIsSignedInUsingAuthV2AThenAccessTokenIsRefreshed() = runTest {
```

Comment:
```suggestion
    fun `when process and remove email success and is signed in using AuthV2 then access token is refreshed`() = runTest {
```
nit, same below

Reply:
I agree that space-separated test method names are slightly easier to read, but I prefer not to mix both approaches inside a single test class. I added a commit that converts all test methods in this class.